sqlglot/expressions.py

```diff
@@ -477,6 +501,39 @@ class Select(Expression):
         "offset": False,
     }
 
+    @classmethod
+    def from_(cls, selectable, selects=None):
```

Comment:
it's annoying that from_ and as_ are reserved keywords, something i hate about pypika + python, any other possibilities here? maybe not...

Reply:
Haha yeah, it's always a bummer.
Either we prepend `_`, which is pretty standard, or come up with a different name.